astrology.lic

```diff
@@ -234,7 +234,12 @@ class Astrology
         bput('open my telescope', 'extend your telescope')
         return observe(body)
       end
-      result = bput('peer telescope', 'You peer aimlessly through your telescope', 'You see nothing regarding the future', 'Roundtime')
+      result = bput('peer telescope', 'You peer aimlessly through your telescope', 'You see nothing regarding the future', 'The pain is too much', 'Roundtime')
+        if result =~ /The pain is too much/
+          bput('stow telescope', 'you put', "That can't be picked up")
+          wait_for_script_to_complete('safe-room')
+          return observe(body)
```

Comment:
Not sure why my last commend is outdated, but i will try commenting again. Would this attempt observe in the healers room?